README.md

```diff
@@ -445,13 +441,21 @@ I would love to receive contributions to this project and welcome PRs from every
 
 ## FAQ
 
-- __My terminal window does not close on selection, even when using `clipse -fc $PPID`__ - _Some terminal environments reference system variables differently. For example, the fish terminal will need to use `$fish_pid` instead. To debug this error you can run `echo $PPID` to see what gets returned. The 'close on selection functionality is also not currently available for macOS as killing the terminals ppid does not close the window - it seems AppleScript is needed to achieve this._
+- __My terminal window does not close on selection, even when using `clipse -fc $PPID`__ 
+<br>
+_Some terminal environments reference system variables differently. For example, the fish terminal will need to use `$fish_pid` instead. To debug this error you can run `echo $PPID` to see what gets returned. The 'close on selection functionality is also not currently available for macOS as killing the terminals ppid does not close the window - it seems AppleScript is needed to achieve this._
 
-- __Is there risk of multiple parallel processes running?__ - _No. The `clipse` command kills any existing TUI processes before opening up and the `clipse -listen`  command kills any existing background listeners before starting a new one._
+- __Is there risk of multiple parallel processes running?__
+<br> 
+_No. The `clipse` command kills any existing TUI processes before opening up and the `clipse -listen`  command kills any existing background listeners before starting a new one._
 
-- __High CPU usage?__ - When an image file has an irregular binary data pattern it can cause a lot of strain on the program when `clipse` reads its history file (even when the TUI is not open). If this happens, you will need to remove the image file from the TUI or buy using `clipse -clear` or `clipse -clear-images` to remove all files if you don't want to spend the time figuring out which one is causing the issue. See issue #33 for an example.
+- __High CPU usage?__ 
+<br>
+When an image file has an irregular binary data pattern it can cause a lot of strain on the program when `clipse` reads its history file (even when the TUI is not open). If this happens, you will need to remove the image file from the TUI or buy using `clipse -clear` or `clipse -clear-images` to remove all files if you don't want to spend the time figuring out which one is causing the issue. See issue #33 for an example.
```

Comment:
```suggestion
When an image file has an irregular binary data pattern, it can cause a lot of strain on the program when `clipse` reads its history file (even when the TUI is not open). If this happens, you will need to remove the image file from the TUI or try using `clipse -clear` or `clipse -clear-images` to remove all files if you don't want to spend the time figuring out which one is causing the issue. See issue #33 for an example.
```